.github/workflows/python.yml

```diff
@@ -125,12 +138,19 @@ jobs:
           ruff format --check .
 
       - name: Run tests
-        if: matrix.version.arch != 'x64-freethreaded'
+        if: matrix.version.arch != 'x64-freethreaded' && matrix.os != 'windows-11-arm'
         run: |
           cargo test
           pip install ".[testing]"
           pytest -sv tests/
 
+      - name: Run tests (Windows arm64)
+        if: matrix.os == 'windows-11-arm'
+        run: |
+          cargo test
+          pip install ".[testing]"
+          pytest -sv tests/ --ignore=tests/test_tf_comparison.py
```

Comment:
I'm skipping the `test_tf_comparison.py` tests here as tensorflow is not available for arm64 windows, and will fail when it tries to import tensor flow. Should I also skip `test_flax_comparison.py`, `test_mlx_comparison.py`, and `test_paddle_comparison.py` here because they will just be all skipped anyways? Or maybe specify just the simple and pt tests like in the free-threaded tests step?